Project/Engine/src/sysFrame/System/Json/Json.cpp

```
@@ -0,0 +1,148 @@
+        const std::filesystem::path& path = entry.path();
+
+        std::string extension = path.extension().string();
+    	if (extension != ".json"){
+            continue;
+        }
+
+        LoadJson(path.stem().string());
+    }
+}
+
+void Json::Save(const std::string& _path) {
```

Cppcheck (2.10-2)

[style] The function 'Save' is never used.

(unusedFunction)